app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -262,12 +264,26 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.kubevirt_password               = data.kubevirt_password;
 
       miqService.sparkleOff();
-
+      $scope.hideDisabledTabs();
       $scope.afterGet  = true;
       $scope.modelCopy = angular.copy( $scope.emsCommonModel );
     }
   };
 
+  $scope.hideDisabledTabs = function() {
+    if ($scope.emsCommonModel.alerts_selection === 'disabled') {
+      $("#alerts_tab").hide();
```

Comment:
Using `angular.element` would be better -

```javascript
angular.element("#alerts_tab").hide();
```

applies to the other `$` occurrences below as well.